articles/analytics/integrations/_install.md

```diff
@@ -0,0 +1,17 @@
+## Install
+
+To add the <%- name %> integration to your app, include a reference to the `Auth0 Analytics.js` script on any pages with Auth0 Lock. Include the script reference after Lock and set the configuration options before the script reference.
+
+```
+<script>
+window.auth0AnalyticsOptions = {
+  // Options
+}
+</script>
+<script src="https://cdn.auth0.com/js/lock/10.x/lock.min.js"></script>
```

Comment:
For Lock (and some other libraries) we use a variable: https://github.com/auth0/docs/blob/master/config/vars.yml#L1

This way with each new release we update only that file. Use it like any other variable ie `${lock_url}`